beacon_node/beacon_chain/src/attestation_verification.rs

```diff
@@ -1035,9 +1035,9 @@ pub fn verify_propagation_slot_range<S: SlotClock, E: EthSpec>(
     attestation: &Attestation<E>,
 ) -> Result<(), Error> {
     let attestation_slot = attestation.data.slot;
-
+    let maximum_gossip_clock_disparity = Duration::from_millis(E::default_spec().maximum_gossip_clock_disparity_millis);
```

Comment:
We cannot use the `default_spec` method here. Reason being parameters in the `ChainSpec` are the ones that can be changed by passing an alternate value in the `config.yml` file. So if we use the `default_spec`, then we would be overwriting a user specified value via the `config.yml` file with the default value. 

In most places, we would have access to the `ChainSpec` item. In this case, the chainspec can be accessed via the `BeaconChain<T>` object (`chain.spec` )